en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/configure-logs-router.md

```diff
@@ -178,4 +178,76 @@ Router to client response headers:
 'x-envoy-decorator-operation', 'localhost:^/v2/pet/findByStatus(\?([^/]+))?$'
 ```
 
-Follow [command line options]({{envoy_path}}/operations/cli) for more information.
\ No newline at end of file
+Follow [command line options]({{envoy_path}}/operations/cli) for more information.
+
+## Interceptor level debug logging
+
+To enable interceptor level debug logging, you need to do the following configuration. 
+
+1. Add `lua:debug` component to the `TRAILING_ARGS` like below:
+```yaml
```

Comment:
Please tab the code blocks so that it comes as a sub-line in the numbered list. If this is not done, the list numbering breaks.